site/guide/model-documentation/view-documentation-activity.qmd

```diff
@@ -20,12 +20,42 @@ This topic matters for model developers, model validators, and auditors.
 
 :::
 
-## Steps
+## Model Activity
```

Comment:
Needs sentence-style caps:

```suggestion
## Model activity
```

Reply:
This one actually reflects what's shown in the UI: the "Model Activity" section (like how we have the "Document Overview").

Reply:
I see — since this isn't clear unless you have the UI side-by-side, I would go with consistency with our style guidelines, but whatever you prefer. Not a biggie either way.